content/en-us/docs/khala-mining/1-1-hardware-requirements.md

```diff
@@ -21,9 +21,12 @@ A system can potentially mine Phala if it meets these general requirements:
    - [Check BIOS compatability](#check-your-bios).
 - [x] A Solid-state drive (SSD) storage device
    - Storing your blockchain data on a mechanical HDD will result in slow syncronosation speeds. At a minimum a 512GB SSD drive is recommended. 
-      - Refer to [issue #554.](https://github.com/Phala-Network/phala-blockchain/issues/554)
+      - Refer to [issue #554](https://github.com/Phala-Network/phala-blockchain/issues/554) for more info.
 - [x] Have a supported version of Ubuntu (18.04, 20.04, 21.04) installed and booted from it.
    - See [Suported OS requirements](#supported-operating-systems). 
+- [x] You require a minimum amount of 5 PHA to mine
+   - Furthermore, you require 2000 PHA tokens to stake. 
```

Comment:
Not exactly 2000. It's based on some calculation

Reply:
Ok thx for the info. A formula might be better to calculate the minimum PHA staking requirements.

Reply:
TODO: - Minimal stake formula: A public Google Sheet for estimation (Suggest to go through the full mining process first, because a lot of factors impact the score)

Reply:
> 

This has now been fixed in this commit: https://github.com/Phala-Network/phala-wiki-next/pull/35/commits/5aa069c47e1f838f667f13dbfc3f756b89bf0c33

A spreadsheet link will be added.